src/frontend/pages/Letters/Letters.tsx

```diff
@@ -98,81 +99,38 @@ export const Letters: React.FunctionComponent<RouteComponentProps> = () => {
     await fetchLetters();
   };
 
-  type OptionType = {
-    value: string | null;
-    label: string;
-  };
-
-  const assigneeOptions: OptionType[] = users
-    .map((u) => ({
-      value: u.uuid as string | null,
-      label: `${u.fullName ? u.fullName + ' - ' : ''}${u.email}`,
-    }))
-    .sort((a, b) => a.label.localeCompare(b.label));
-
-  const handleUserSelection = (letter: ApiLetterAdmin, opt: OptionType | null) => {
-    if (opt && opt.value !== letter.assignedResponderUuid) {
-      assignLetter({ letterUuid: letter.uuid, assigneeUuid: opt.value });
-    } else {
-      // Unassigned letter will set assigneeUuid field to null
-      assignLetter({ letterUuid: letter.uuid, assigneeUuid: null });
-    }
-  };
+  const filteredLetters = letters.filter((letter) =>
+    isDateInActiveDateRanges(moment(letter.created), [selectedDateRange]),
```

Comment:
Now I see it in action, I think we should still include the letter created on the end date.